core/indigo-core/molecule/molecule_json_saver.h

```diff
@@ -43,6 +43,7 @@ namespace indigo
         void saveMolecule(BaseMolecule& bmol);
         void saveMolecule(BaseMolecule& bmol, rapidjson::Writer<rapidjson::StringBuffer>& writer);
         bool _add_stereo_desc;
+        static void saveSimpleObjects(const PtrArray<MetaObject>& meta_objects, rapidjson::Writer<rapidjson::StringBuffer>& writer);
```

Comment:
For the sake of symmetricity, it's better use `MetaObjectsInterface&` here - the same typ[e as in `loadSimpleObjects`.

Reply:
fixed